### PR TITLE
Update for Italian translation

### PR DESCRIPTION
**Pull request by Mauron**
Translation update according to file updates until May 12th 2015.

### CODE REVIEW

**Comment by javiexin** [comment]
Please, take a look at this Travis error:

FILE: .../phpBB3/phpBB/ext/wolfsblvt/advancedpolls/language/it/advancedpolls.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 64 | ERROR | Whitespace found at end of line
    |       | (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
--------------------------------------------------------------------------------

**Comment by Mauron** [comment]
I guess that should do the trick now.
I'm impressed that such a thing could throw out an error and interrupt the validation process.

**Comment by javiexin** [comment]
Thanks

**Comment by javiexin** [comment]
Please, update the IT translation for Advanced Polls, new strings are required.

**Comment by Mauron** [comment]
There we go!

File: language/it/info_acp_advancedpolls.php [hunk 1]
```diff
@@ -51,9 +53,24 @@
...
+	'AP_ACT_POLL_SCORING_EXPLAIN'	=> 'Attiva l’opzione per assegnare punteggi differenti alle opzioni di voto.',
+	'AP_ACT_INCREMENTAL_VOTES'		=> 'Attiva voto incrementale',
+	'AP_ACT_INCREMENTAL_VOTES_EXPLAIN'	=> 'Attiva l’opzione per votare in maniera incrementale, fintanto che non sia estinta la propria possibilità di voto.',
+	'AP_ACT_CLOSED_VOTING'			=> 'Attiva voto chiuso',
+	'AP_ACT_CLOSED_VOTING_EXPLAIN'	=> 'Attiva l’opzione per permettere il voto in sondaggi aperti in topic chiusi.',
+	'AP_ACT_POLL_END'				=> 'Attiva termine voto',
+	'AP_ACT_POLL_END_EXPLAIN'		=> 'Attiva l’opzione per specificate la data e/o l’ora di fine sondaggio, invece di specificarne la durata a partire dall’inizio del sondaggio.',
+	'AP_ACT_POLL_NOTIFICATIONS'				=> 'Attiva notifiche sondaggio',
+	'AP_ACT_POLL_NOTIFICATIONS_EXPLAIN'		=> 'Attiva l’invio di notifiche ai votanti alla scadenza di una votazione con voti nascosti per cui sono visibili i risultati.',
+
```
Comment (javiexin) [inline language/it/info_acp_advancedpolls.php]: ----------------------------------------------------------------------
 70 | ERROR | [x] Whitespace found at end of line
    |       |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
----------------------------------------------------------------------

**Comment by javiexin** [comment]
Please, update the Advanced Polls translation.  There are a few new strings (4 or 5) in advancedpolls.php.

**Comment by Mauron** [comment]
I have to open new PR, do I?

**Comment by javiexin** [comment]
Sure, this one is closed.  But as I cannot send you a PM, I had to contact you this way :)

**Comment by Mauron** [comment]
No problem, here it goes!

**Pull request merged**